js/functions.js

```diff
@@ -0,0 +1,47 @@
+// Функция для проверки длины строки
+
+function checksStringLength(string, number) {
+  return (string.length <= number);
+}
+
+checksStringLength('I like JavaScript', 22);
+
+
+// Функция для проверки, является ли строка палиндромом 1й способ.
+
+
+function palindromeChecker(string) {
```

Comment:
Лучше называть глаголами
```suggestion
function isPalindrome(string) {
```